psydac/api/ast/evaluation.py

```diff
@@ -456,7 +460,7 @@ def _initialize(self):
         body =  init_vals + body
 
         func_args = self.build_arguments(degrees + basis + fields_coeffs + fields_val)
-
+        
```

Comment:
@saidctb Please remove any leading white spaces from the empty lines...

Reply:
it's fixed